pyodide-build/pyodide_build/tests/test_cli.py

```diff
@@ -328,3 +335,24 @@ def test_create_zipfile_compile(temp_python_lib, tmp_path):
     with ZipFile(output) as zf:
         assert "module1.pyc" in zf.namelist()
         assert "module2.pyc" in zf.namelist()
+
+
+def test_xbuildenv_install(tmp_path):
+    envpath = Path(tmp_path) / ".xbuildenv"
+    result = runner.invoke(
+        xbuildenv.app,
+        [
+            "install",
+            "--path",
+            str(envpath),
+            "--download",
+            "--url",
+            "https://github.com/pyodide/pyodide/releases/download/0.22.1/xbuildenv-0.22.1.tar.bz2",
```

Comment:
It's a 10MB file, not sure about downloading it each time this test is run. At least maybe we could put it behind some pytest mark and not run it by default?

If we cannot create a similar archive for testing at runtime that is.

Reply:
> If we cannot create a similar archive for testing at runtime that is.

I'll try this approach.